components/camel-hdfs/src/main/java/org/apache/camel/component/hdfs/HdfsConfiguration.java

```diff
@@ -86,6 +86,9 @@
     @UriParam
     private String owner;
 
+    @UriParam(label = "consumer,experimental", defaultValue = "false")
```

Comment:
I don't think we use experimental label anywhere in the project, consumer should be enough

Reply:
Removed.
I've used the _experimental_ label because I haven't tested this for all scenarios.